org.dash.electrum.electrum_dash.json

```diff
@@ -0,0 +1,63 @@
+{
+    "id": "org.dash.electrum.electrum_dash",
+    "sdk": "org.kde.Sdk",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "rename-icon": "electrum-dash",
+    "rename-desktop-file": "electrum-dash.desktop",
+    "command": "electrum-dash",
+    "finish-args": [
+        /* Wayland and Xorg support */
+        "--socket=x11",
+        "--socket=wayland",
+        /* Sound */
+        "--socket=pulseaudio",
+        /* Persistent storage */
+        "--persist=.electrum-dash",
+        /* OpenGL access. */
+        "--device=dri",
+        "--share=ipc",
+        /* Network acess. */
+        "--share=network"
+    ],
+    "modules": [
+        "libsecp256k1.json",
+        "python3-modules.json",
+        "python3-pycryptodomex.json",
+        "python3-pyQt5.json",
+        "libusb.json",
+        "python3-cython.json",
+        "python3-hidapi.json",
+        "python3-trezor_agent.json",
+        "python3-x11-hash.json",
+        "python3-bls.json",
+        {
+            "name": "electrum-dash",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} .[fast]",
+                "install -Dp -m 644 org.dash.electrum.electrum_dash.appdata.xml /app/share/metainfo/org.dash.electrum.electrum_dash.appdata.xml",
+                "install -Dp -m 644 electrum_dash/gui/icons/electrum-dash.png /app/share/icons/hicolor/128x128/apps/electrum-dash.png"
+            ],
+            "post-install": [
+                "desktop-file-install --dir=/app/share/applications --set-key=Exec --set-value=electrum-dash --set-key=Icon --set-value=org.dash.electrum.electrum_dash electrum-dash.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/akhavr/electrum-dash.git",
+                    "tag": "3.3.8.5",
+                    "commit": "b52b5edef6263226ef9dd7cdcea815147408cde5"
+                },
+                {
+                    "type": "patch",
+                    "path": "electrum-dash-setup.patch"
+                },
+                {
+                    "type": "file",
+                    "path": "org.dash.electrum.electrum_dash.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I read what needs to be fixed.

Reply:
added some fixes with 0763cde:
- upgrade appdata to metainfo
- change screenshots
- add translate url
- build from sdist archive to include translations